components/Navigation/Navigation.constants.tsx

```diff
@@ -8,26 +8,26 @@ export const NAVIGATION_ICONS: INavigationIcon[] = [
   },
   {
     icon: 'search',
-    href: '/search',
+    href: 'search',
```

Comment:
This PR has changes from another branches and issues. Please create a new branch, cherry pick your changes and create another PR